Dockerfiles/ci/README.md

```diff
@@ -0,0 +1,37 @@
+# CVP upstream image for testing in github actions
+CVP upstream image operator bundle validate container
```

Comment:
Could we make this more descriptive? Like "This readme describes how to build and execute a container that validates an upstream operator bundle."